website/docs/r/api_management_api_tag_description.html.markdown

```diff
@@ -0,0 +1,106 @@
+---
+subcategory: "API Management"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_api_management_api_tag_description"
+description: |-
+  Manages an API Tag Description within an API Management Service.
+---
+
+# azurerm_api_management_api_tag_description
+
+Manages an API Tag Description within an API Management Service.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_api_management" "example" {
+  name                = "example-apim"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  publisher_name      = "My Company"
+  publisher_email     = "company@terraform.io"
+
+  sku_name = "Developer_1"
+}
+
+resource "azurerm_api_management_api" "example" {
+  name                = "example-api"
+  resource_group_name = azurerm_resource_group.example.name
+  api_management_name = azurerm_api_management.example.name
+  revision            = "1"
+  display_name        = "Example API"
+  path                = "example"
+  protocols           = ["https"]
+
+  import {
+    content_format = "swagger-link-json"
+    content_value  = "http://conferenceapi.azurewebsites.net/?format=json"
+  }
+}
+
+resource "azurerm_api_management_tag" "example" {
+  api_management_id = azurerm_api_management.example.id
+  name              = "example-Tag"
+}
+
+resource "azurerm_api_management_api_tag_description" "example" {
+  api_name                  = azurerm_api_management_api.example.name
+  api_management_name       = azurerm_api_management.example.name
+  resource_group_name       = azurerm_resource_group.example.name
+  tag_name                  = azurerm_api_management_tag.example.name
+  description               = "This is an example description"
+  external_docs_url         = "https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs"
+  external_docs_description = "This is an example external docs description"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `api_name` - (Required) The name of the API Management API. Changing this forces a new resource to be created.
+
+* `api_management_name` - (Required) The Name of the API Management Service. Changing this forces a new resource to be created.
+  
+* `resource_group_name` - (Required) The Name of the Resource Group in which the API Management Service exists. Changing this forces a new resource to be created.
+
+* `tag_name` - (Required) The name of the tag. It must be known in the API Management instance. Changing this forces a new API Management API Tag to be created.
```

Comment:
couldn't this all be replaced by `tage_id = azurerm_api_management_tag.example.id` ?

Reply:
My mistake, this property is different from `tag_id`,  it should named with `tag_desc_id` and `api_name`, `resource_group_name` etc. can be  replaced by `api_id`.
https://docs.microsoft.com/en-us/rest/api/apimanagement/current-ga/api-tag-description/create-or-update?tabs=HTTP#uri-parameters